src/tir/analysis/block_access_region_detector.cc

```diff
@@ -181,6 +181,34 @@ void BlockReadWriteDetector::VisitStmt_(const IfThenElseNode* op) {
 }
 
 void BlockReadWriteDetector::VisitExpr_(const CallNode* op) {
+  if (op->op.same_as(builtin::tvm_access_ptr())) {
```

Comment:
Please add testcase for `BlockReadWriteDetector` directly in `tests/python/unittest/test_tir_analysis_get_block_access_region.py`

Reply:
Should I keep the testcase in tests/python/unittest/test_tir_schedule_compute_inline.py?

Reply:
Of course, the more test cases the better